framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -205,6 +206,12 @@
      */
     private final Set<Class<? extends Annotation>> supportedQuals;
 
+    /**
+     * To cache the supported type qualifier names. call {@link #getSupportedTypeQualifierNames()}
```

Comment:
Can you document that these are the fully-qualified names, not the simple names.